### DISCUSSION TITLE
关于 chrome 游览器

### DISCUSSION THREAD

**Discussion by ths0616** [the1812/Bilibili-Evolved#3004] [功能建议 / Ideas]
在chrome游览器，bilibili播放页面默认【打开高能进度条】以及默认【自动】画质
十分影响观影体验，能否添加关于【关闭高能进度条】，以及添加【默认最高画质】这一插件
![image](https://user-images.githubusercontent.com/76588284/153771587-34c9f264-e75b-427b-9c88-57bafe451dfb.png)

**Comment by Geeyun-JY3** [comment]
b 站现在会记忆画质：https://github.com/the1812/Bilibili-Evolved/issues/1274#issuecomment-739481607

点击高能进度条右边的钉子📌图标就能关闭。
